controller/workitem.go

```diff
@@ -246,11 +246,17 @@ func (c *WorkitemController) Reorder(ctx *app.ReorderWorkitemContext) error {
 				return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, "failed to reorder work item"))
 			}
 
+			// check if the workitems to reorder belongs to the space
+			_, err = appl.WorkItems().Load(ctx, ctx.SpaceID, wi.Number)
+			if err != nil {
+				return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, "failed to reorder work item"))
```

Comment:
@hectorj2f changed the error message as per @baijum 's suggestion -. https://github.com/fabric8-services/fabric8-wit/pull/1478/files#diff-02845b093c442bf1ae790a6ebb6c8da8R252